C/common/CMakeLists.txt

```diff
@@ -35,7 +39,7 @@ include_directories(include ../services/common/include ../common/include ../thir
 if(${CMAKE_VERSION} VERSION_LESS "3.12.0")
     include_directories(${PYTHON_INCLUDE_DIRS})
 else()
-    include_directories(${Python_INCLUDE_DIRS})
+    include_directories(${Python_INCLUDE_DIRS} ${Python_NUMPY_INCLUDE_DIRS})
```

Comment:
Would it not be better to use 

target_link_libraries(${PROJECT_NAME} ${Python_LIBRARIES} Python3::NumPy)

https://stackoverflow.com/questions/59012911/better-way-to-include-a-directory-in-cmakelists-txt

Reply:
do we still need ${Python_NUMPY_INCLUDE_DIRS}?